content-src/lib/first-run-data.js

```diff
@@ -0,0 +1,90 @@
+const IMAGE_PATH = "img/";
+const FIRST_RUN_TYPE = "first-run";
+const FAVICON_PATH = "favicons/images/";
+
+module.exports = {
+  FIRST_RUN_TYPE,
+  TopSites: [
```

Comment:
Although requires some additional work on tippy-top, and maybe should follow our tippy-top implementation, so I might fiel a follow-up for this